src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -713,3 +713,26 @@ g.test('multisample_render_target_formats_support_resolve')
       colorAttachments: [colorAttachment],
     });
   });
+
+g.test('occlusion_query_set_type')
```

Comment:
nit: use "occlusionQuerySet" because that's the name of the field being tested
```suggestion
g.test('occlusionQuerySet,type')
```

Reply:
Or `occlusionQuerySet,type_and_valid` if testing both